server/image_pull.go

```diff
@@ -133,7 +132,12 @@ func (s *Server) pullImage(ctx context.Context, pullArgs *pullArguments) (string
 	ctx, span := log.StartSpan(ctx)
 	defer span.End()
 
-	sourceCtx := *s.config.SystemContext   // A shallow copy we can modify
+	sourceCtx, err := s.contextForNamespace(pullArgs.namespace)
```

Comment:
`contextForNamespace()` wraps the namespace check below. Should we drop the lines around? https://github.com/cri-o/cri-o/blob/73b6f0dbc6da5c349aacb8393f0b33143325a694/server/image_pull.go#L142

Reply:
For clarity, this is the diff: https://github.com/harche/cri-o/compare/sigstore_namespace...QiWang19:cri-o:pr?expand=1

Reply:
/cc @saschagrunert WDYT?

Reply:
Yes let's simplify that.